src/TokenLoader.tsx

```diff
@@ -42,37 +42,22 @@ export default class TokenLoader extends Component<TokenLoaderProps, TokenLoader
       return;
     }
 
-    if(!this.props.location.search){
+    if(!this.props.location.hash){
       this.setState({forwardUrl: this.props.location.pathname});
       return;
     }
   
-    const queries = queryString.parse(this.props.location.search);
+    const tokens = queryString.parse(this.props.location.hash);
 
-    if(queries.meetingtoken){
-      this.saveMeetingToken.Execute(queries.meetingtoken as string);
-      delete queries.meetingtoken;
+    if(tokens.officerToken){
+      this.saveMeetingToken.Execute(tokens.officerToken as string);
     }
-    else if(queries.token){
-      this.saveSignOffToken.Execute(queries.token as string);
-      delete queries.token;
+    
+    if(tokens.traToken){
+      this.saveSignOffToken.Execute(tokens.traToken as string);
```

Comment:
likewise, saveTraToken

Reply:
Can definitely change the names in the loader - I'm hesitant to change the names of the UseCases and Gateway methods just because I'm terrified of something going wrong at this stage, but I'll do it if you think it's best! :smile: